packages/dds/tree/src/feature-libraries/chunked-forest/codec/codecs.ts

```diff
@@ -136,7 +136,7 @@ function clientVersionToFieldBatchVersion(
 	return brand(
 		getConfigForMinVersionForCollab(clientVersion, {
 			[lowestMinVersionForCollab]: FieldBatchFormatVersion.v1,
-			[FluidClientVersion.v2_73]: FieldBatchFormatVersion.v2,
+			[FluidClientVersion.v2_74]: FieldBatchFormatVersion.v2,
```

Comment:
Isn't that change going to cause a lot of confusion: some clients running with FF 2.73 will potentially use FieldBatchFormatVersion.v2, while some client running with FF 2.74 will have this logic which says FieldBatchFormatVersion.v2 shouldn't be used before 2.74.